pkg/subsystem/list.go

```diff
@@ -28,5 +28,8 @@ func GetList(name string) []*Subsystem {
 }
 
 func ListService(name string) *Service {
+	if len(lists[name]) == 0 {
+		panic(name + " subsystem list is empty!")
```

Comment:
Must version must already panic.
Non-Must version must not panic.

Reply:
Moved to `MustMakeService`. PTAL.